phone/Dial.py

```diff
@@ -0,0 +1,55 @@
+from gpiozero import Button
+from threading import Timer
+
+
+class Dial:
+  def __init__(self, port_cycle, port_signal):
+    self.__button_cycle = Button(port_cycle)
+    self.__button_cycle.when_pressed = self.__dial_cycle_pressed
+    self.__button_cycle.when_released = self.__dial_cycle_released
+
+    self.__button_signal = Button(port_signal)
+    self.__button_signal.when_pressed = self.__dial_signal_pressed
+
+    self.__signal_count = None
+    self.__timeout = None
+    self.__numbers = []
+  
+  DIAL_TIMEOUT = 1.5
+
+  def when_number_dialled(self, callback):
+    self.__when_number_dialled_callback = callback
```

Comment:
Does it make sense to use [events](https://pypi.python.org/pypi/Events) instead?